packages/dashboard/src/PanelManager.ts

```diff
@@ -271,6 +301,79 @@ class PanelManager {
     this.sendUpdate();
   }
 
+  handleCycleStack(direction: CycleDirection): void {
```

Comment:
This function isn't being used as an event handler (e.g. it's not being passed directly to `eventHub.on` or another event emitter, and you don't need to `.bind` it).
Just call this function `cycleStack` instead. We use the `prefix` handle specifically for event handlers, to match with React's guide: https://react.dev/learn/responding-to-events#adding-event-handlers
> Event handler functions:    
> - Are usually defined inside your components.
> - Have names that start with handle, followed by the name of the event.

```suggestion
  cycleStack(direction: CycleDirection): void {
```